packages/react-utils/src/components/GenericDetailsView/ResourceDetails/index.tsx

```diff
@@ -33,7 +35,10 @@ export const ResourceDetails = (props: ResourceDetailsProps) => {
     footer,
     status,
     headerLeftDataClasses,
+    theme = 'default',
+    column = 2,  // Default column value
```

Comment:
Can we use the default type of Description component column prop here. We might need to have responsiveness which can't be achieved by using this as number.